lib/screens/profile.dart

```diff
@@ -0,0 +1,217 @@
+                        child: Container(
+                          height: 40,
+                          width: 40,
+                          child: Icon(
+                            Icons.add_a_photo,
+                            color: Colors.white,
+                          ),
+                          decoration: BoxDecoration(
+                              color: Colors.grey,
+                              borderRadius:
+                                  BorderRadius.all(Radius.circular(20))),
+                        ))
+                  ],
+                ),
+                Form(
+                  key: _formKeyName,
+                  child: Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: ExpansionTile(
+                      title: Row(
+                        children: [
+                          Text(ShipantherLocalizations.of(context).username),
+                          Text(_updatedName == null
+                              ? widget.user.name
+                              : _updatedName),
+                        ],
+                      ),
+                      trailing: Icon(Icons.edit),
+                      childrenPadding: EdgeInsets.only(left: 10, right: 10),
+                      children: [
+                        Column(
+                          children: [
+                            TextFormField(
+                              initialValue: _updatedName == null
+                                  ? widget.user.name
+                                  : _updatedName,
+                              decoration: InputDecoration(
+                                  labelText:
+                                      ShipantherLocalizations.of(context).name),
+                              autocorrect: false,
+                              enableSuggestions: false,
+                              keyboardType: TextInputType.text,
+                              validator: (String value) {
+                                if (value.isEmpty) {
+                                  return ShipantherLocalizations.of(context)
+                                      .paramRequired(
+                                          ShipantherLocalizations.of(context)
+                                              .password);
+                                }
+
+                                return null;
+                              },
+                              onSaved: (val) =>
+                                  setState(() => _updatedName = val),
+                            ),
+                            Container(
+                              padding:
+                                  const EdgeInsets.symmetric(vertical: 16.0),
+                              alignment: Alignment.center,
+                              child: SignInButtonBuilder(
+                                icon: Icons.save,
+                                backgroundColor: Colors.blue,
```

Comment:
Buttons should follow a consistent theme throughout the app. We don't want to put colors all over the place